2.train_model/README.md

```diff
@@ -2,7 +2,10 @@
 
 In this module, we train ML models to predict phenotypic class from cell features.
```

Comment:
```suggestion
In this module, we trained ML models to predict phenotypic classes from cell features.
```

Reply:
I think the tense I used is correct here... maybe @gwaybio can confirm? If this isn't correct I will have to address it in the rest of the modules as well.

I remember when I settled for this tense and voice I was following the [Cell Health](https://github.com/broadinstitute/cell-health/tree/master/0.download-data) example. For example:
> In this module, we download all single-cell Cell Painting profiles generated for the Cell Health project.

Reply:
Consistency is the most important. I tend to prefer present tense in analysis repositories, but past tense in papers.